lib/node_modules/@stdlib/stats/base/dists/logistic/ctor/benchmark/benchmark.js

```diff
@@ -414,11 +494,15 @@ bench( pkg+':logcdf', function benchmark( bm ) {
 	mu = 1.0;
 	s = 2.0;
 	dist = new Logistic( mu, s );
+	len = 100;
+	x = new Float64Array(len);
+	for (i = 0; i < len; i++) {
+		x[ i ] = uniform( 0.0, 1.0 );
```

Comment:
```suggestion
		x[ i ] = uniform( -3.0, 3.0 );
```

The range should remain the same.